backend/src/main/java/corea/matching/domain/MatchResult.java

```diff
@@ -37,4 +37,7 @@ public class MatchResult {
     public MatchResult(long roomId, Member fromMember, Member toMember, String prLink) {
         this(null, roomId, fromMember, toMember, prLink, ReviewStatus.INCOMPLETE);
     }
+    public void reviewComplete(){
```

Comment:
[수정 요청] 개행 + 스페이스바 

[제안] 메서드명?

```suggestion
    
    public void reviewCompleted() {
```

Reply:
해당 부분에 대해서 저도 생각했는데
Review 도메인이라면 complete 만 명시하겠으나
MatchResult 의 complte 라면, `매칭 결과 가 완료되었구나` 라고 생각할 수도 있겠다고 생각해서 저렇게 명시했습니당.